ui/ui-frontend/projects/collect/src/assets/i18n/en.json

```diff
@@ -68,7 +68,7 @@
     },
     "TITLE_SEARCH": "Search : ",
     "TITLE_SEARCH_AREA": "Searching by title or description",
-    "TITLE_TRANSACTIONS_MONITORING": "Transaction monitoring",
+    "TITLE_TRANSACTIONS_MONITORING": "Transaction monitoring of project",
```

Comment:
ça été ajouté en anglais?

Reply:
cela est en anglais ^^
en fait il existe déjà j'ai ajouté juste of project